operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/ControllerConfiguration.java

```diff
@@ -51,15 +56,28 @@
   String labelSelector() default Constants.NO_VALUE_SET;
 
   /**
+   * Use onAddFilter, onUpdateFilter, onDeleteFilter instead.
+   *
    * <p>
    * Resource event filters only applies on events of the main custom resource. Not on events from
    * other event sources nor the periodic events.
    * </p>
    *
    * @return the list of event filters.
    */
+  @Deprecated
   Class<? extends ResourceEventFilter>[] eventFilters() default {};
 
+  /**
+   * Filter of onAdd events of resources. (Note that onDelete is missing, delete events don't
```

Comment:
Hmm, kk, actually was thinking if this needs explicit docs, since it is quite straightforward, so javadoc might be enough. But had to put the delete part somewhere. Can remove it, and maybe just add an FAQ?

Reply:
removed from here.